WcaOnRails/db/migrate/20170816143703_add_greater_china_championship_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddGreaterChinaChampionshipType < ActiveRecord::Migration[5.1]
+  def change
+    %w(CN HK MO TW).each do |iso2|
+      EligibleCountryIso2ForChampionship.create!(championship_type: "greater_china", eligible_country_iso2: iso2)
```

Comment:
Aha. That does make sense. You could make the app generate a translation key, but since the translations need to be in `git` we have to make PRs for each multi competition anyway. So this is fine.

So the UI part with translatable text will be in a future PR?

Thanx for the clarification, BTW. Part of me wondered if this was a way to stop people from creating unauthorized multi-competitions, while anyone could mark their event as a World Championship :)